src/drivers/scsi/scsi_disk_create.c

```diff
@@ -52,6 +31,8 @@ static void *scsi_create_thread(void *arg) {
 	assert(arg);
 	sdev = arg;
 
+	mutex_init(&sdev->m);
```

Comment:
do we use a thread now?

Reply:
Nope, will try to remove

Reply:
Hmm, it looks like these mutexes are used for the case when multiple threads can use SCSI device. So currently it is not a valid situation, of course, but the code with mutexes lock/unlock  looks like to be valid, so I am not whether to remove it...